### PR TITLE
removed jquery

### PR DESCRIPTION
**Pull request by BeastJsWeb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,26 @@
-# jQuerySimpleCart
+# SimpleCart
 Simple order cart for your Site
 
 ### How to integrate
-	1. Copy the layout from cart.html and put it somewhere into your site (need jQuery to be included first)
+	1. Copy the layout from cart.html and put it somewhere into your site
 	2. Configure this part for link it to products:
```
Comment (molotow11) [inline README.md:7]: Please specify where this should be configured?

File: cart.html [hunk 2]
```diff
@@ -56,28 +55,31 @@
...
-<div class="jQuerySimpleCartDetails">
-	<div class="close">x</div>
-	<div class="jQuerySimpleCartDetails-title">Товары в корзине</div>
-	<ul class="products"></ul>
-	<div class="cartTotals">
-		Всего продуктов <span class="productsCount">0</span> на сумму <span class="totalPrice">0</span> Р
-		<button class="btn btn-success place-order">Сделать заказ</button>
-		<a href="#" class="btn clearCart">Очистить корзину</a>
+<div id="SimpleCart">
+	<div class="SimpleCart-empty">Ваша корзина пуста</div>
```
Comment (molotow11) [inline cart.html]: Can we translate in English maybe?

File: cart.html [hunk 3]
```diff
@@ -88,246 +90,17 @@
...
-				var c = ca[i];
-				while (c.charAt(0)==' ') c = c.substring(1,c.length);
-				if (c.indexOf(nameEQ) == 0) return c.substring(nameEQ.length,c.length);
-			}
-			return null;
-		}
-	});			
-</script>
\ No newline at end of file
+<script type="text/javascript" src="cart.js"></script>
```
Comment (molotow11) [inline cart.html]: Please rename all the files in some specific, like `simple-cart` etc..

File: demo/SimpleCart.js [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+
+    //clear cart button
+    cartDetailsClear.addEventListener("click", () => {
+      clearCookie({
+        name: cookieProductsName,
+        callback: buildCartLayout,
+      });
+    });
+  }
+});
```
Comment (molotow11) [inline demo/SimpleCart.js]: Did you checked it all and works fine?

**Review by molotow11 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BeastJsWeb) [reply README.md:7]: ok

File: cart.html (hunk 2 above)
Comment (BeastJsWeb) [reply cart.html]: sure

File: cart.html (hunk 3 above)
Comment (BeastJsWeb) [reply cart.html]: ok

File: demo/SimpleCart.js (hunk 4 above)
Comment (BeastJsWeb) [reply demo/SimpleCart.js]: yes

**Comment by molotow11** [comment]
@BeastJsWeb Great! Can we add demo page here somehow? github pages etc.?

**Comment by BeastJsWeb** [comment]
> @BeastJsWebБольшой! Можем ли мы как-то добавить сюда демо-страницу? страницы github и т. д.?

Sure, I think so

**Comment by molotow11** [comment]
@BeastJsWeb Ok, I will merge for now. Please propose fixes for add demo page

**Review by molotow11 — APPROVED** [review]

**Pull request merged**